membership-common/src/test/scala/com/gu/memsub/BillingScheduleTest.scala

```diff
@@ -179,30 +176,26 @@ class BillingScheduleTest extends Specification {
     "Include any products found with the productFinder in the bill items" in {
```

Comment:
not sure whether to delete this test as there's no product finder (i.e. catalog) any more